src/sentry/api/endpoints/group_external_issues.py

```diff
@@ -1,13 +1,23 @@
+import logging
+
 from sentry.api.bases.group import GroupEndpoint
 from sentry.api.serializers import serialize
 from sentry.models import PlatformExternalIssue
 
+logger = logging.getLogger("sentry.group_external_issues.api")
```

Comment:
I think you can just use `"sentry.api"` since `group_external_issue` is in the message.